base_partner_sequence/tests/test_base_partner_sequence.py

```diff
@@ -7,29 +7,49 @@
 
 
 class TestBasePartnerSequence(common.TransactionCase):
-
     def setUp(self):
         super(TestBasePartnerSequence, self).setUp()
 
         self.res_partner = self.env['res.partner']
-        self.partner = self.res_partner.create({
```

Comment:
Don't do stylistic changes like this one. Always keep diff minimum if possible.

Reply:
My bad, automatic style manager, fixed